src/main.cpp

```
@@ -405,21 +437,21 @@ bool SMPDebug_Execute(
 	}
 
 	if (_strnicmp(buffer, "off", MAX_PATH) == 0) {
 		hdt::SkyrimPhysicsWorld::get()->disabled = true;
 		{
-			RE::ConsoleLog::GetSingleton()->Print("HDT-SMP disabled");
+			hdt::smpEcho("HDT-SMP disabled");
 		}
 		return true;
 	}
 
 	if (_strnicmp(buffer, "QueryOverride", MAX_PATH) == 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)